### ISSUE TITLE
Fail to update to 0.67.1

### ISSUE THREAD

**Issue by mbarbero** [jbangdev/homebrew-tap#5]
Today, my `brew update` broke with

```
❯ brew upgrade
...
==> Upgrading jbangdev/tap/jbang 0.66.1 -> 0.67.1
==> Downloading https://github.com/jbangdev/jbang/releases/download/v0.67.1/jbang-0.67.1.zip
Already downloaded: /Users/mbarbero/Library/Caches/Homebrew/downloads/2f5e6e074832b3443de31177a0f816a66b94a4fec1bae024faff9153dee81b77--jbang-0.67.1.zip
Error: An exception occurred within a child process:
  Utils::Inreplace::Error: inreplace failed
/usr/local/Cellar/jbang/0.67.1/libexec/bin/jbang:
  expected replacement of /^abs_jbang_path=.*/ with "abs_jbang_path=/usr/local/Cellar/jbang/0.67.1/libexec/bin/jbang.jar"
```

Same error when uninstalling and re-installing: 

```
❯ brew remove jbang
Uninstalling /usr/local/Cellar/jbang/0.66.1... (7 files, 5.8MB)
❯ brew install jbang
...
==> Installing jbang from jbangdev/tap
==> Downloading https://github.com/jbangdev/jbang/releases/download/v0.67.1/jbang-0.67.1.zip
Already downloaded: /Users/mbarbero/Library/Caches/Homebrew/downloads/2f5e6e074832b3443de31177a0f816a66b94a4fec1bae024faff9153dee81b77--jbang-0.67.1.zip
Error: An exception occurred within a child process:
  Utils::Inreplace::Error: inreplace failed
/usr/local/Cellar/jbang/0.67.1/libexec/bin/jbang:
  expected replacement of /^abs_jbang_path=.*/ with "abs_jbang_path=/usr/local/Cellar/jbang/0.67.1/libexec/bin/jbang.jar"
```

Is this something happening on my machine only? Do you a suggestion?

**Comment by maxandersen** [comment]
have past updates worked for you?

we didn't change the brew script in a loong time so the issue surprises me.

**Comment by mbarbero** [comment]
Yep, all past updates worked for me.

**Comment by maxandersen** [comment]
fixed in 0.67.2

**Comment by mbarbero** [comment]
For future reference, fixed in https://github.com/jbangdev/jbang/commit/27c602ed1890f74e503884f78ad351d1c58a7e9c (caused by https://github.com/jbangdev/jbang/pull/747)
